ember.md

```diff
@@ -183,3 +185,25 @@ CSS is permitted (and encouraged) in apps and addons under certain circumstances
 </div>
 
 ```
+<a name="testing"></a>
+## Testing
+
+<a name="testing--test-scripts"></a>
+### 5.1 Test Scripts
+
+> Why? For consistency in testing across all of our Ember apps
```

Comment:
I think an even more powerful point is: we want meaningful tests and to ensure that those tests pass whenever we build a new version of an app.